test/transpile.jl

```diff
@@ -962,3 +962,11 @@ end
     @test !circuit_contains_gate_type(transpiled_circuit, Snowflake.Swap)
     @test simulate(circuit) ≈ simulate(transpiled_circuit)
 end
+
+@testset "CastControlledGatesToNativeTranspiler" begin
```

Comment:
Native gates depend on the QPU and shouldn't be part of the transpiler name. I think renaming it to `UnsupportedGatesTranspiler` would be more apt, or any other name that does not make assumptions about the QPU.

Reply:
name changed